doc/api/corepack.md

```diff
@@ -64,6 +64,12 @@ package manager version you wish to set:
 corepack prepare yarn@x.y.z --activate
 ```
 
+Alternately, a [`dist-tag`](https://docs.npmjs.com/downloading-and-installing-packages-locally#installing-a-package-with-dist-tags) may be used:
+
+```bash
+corepack prepare yarn@latest --activate
```

Comment:
```suggestion
corepack prepare yarn@stable --activate
```
`@latest` is not a valid tag for `yarn`.

Reply:
What do you mean? `@latest` is always a valid tag on npm (it cannot be deleted).

Reply:
Yarn Berry is not on npm:

```console
$ corepack prepare yarn@latest           
(node:96490) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Usage Error: Tag not found (latest)

$ corepack prepare yarn@stable
(node:96513) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Preparing yarn@stable...
```

Reply:
In that case, it would be confusing to point to the npmjs documentation about dist-tags, no?

Reply:
@arcanis is this something we can fix in the list of tags?

Reply:
Re: yarn tag / version weirdness: Likely a holdout from the protracted move from v1 --> v2, when the recommended way to install v2 was to first update v1, then to run `yarn set version berry` in the project -- you can still see that flow in the [migration guide](https://yarnpkg.com/getting-started/migration#step-by-step). I believe the intention was to prevent headaches for those on v1.

Reply:
I can change the example to use pnpm. Kinda tempted to change all three yarn commands in the "Upgrading the global versions" section to pnpm, so it doesn't seem random when it suddenly jumps from yarn to pnpm. Does that sound right?

Reply:
We can also have several examples, e.g.:

````markdown
```bash
corepack prepare pnpm@latest --activate
corepack prepare yarn@stable --activate
```
````

Reply:
@aduh95, I like that ("several examples"), just pushed a commit to change it to that.

FYI, when I run prepare with yarn stable & latest (node v18.9.0, corepack 0.14.0, starting with Yarn 3.2.3), I get:
`corepack prepare yarn@stable --activate` --> `Preparing yarn@stable for immediate activation...`, still on Yarn 3.2.3 after.
`corepack prepare yarn@latest --activate` --> `Usage Error: Tag not found (latest)`, still on Yarn 3.2.3 after (of course).

Reply:
Changed it to say "a tag or range may be used" instead of "a `dist-tag` may be used", since `dist-tag` is rarely specified in docs for npm, Yarn or pnpm. I think I was being too specific / low-level. Dropped the link as well.

Tempted to link to https://github.com/npm/node-semver#ranges, that's where [/cli/v8/configuring-npm/package-json#dependencies](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) links to.

There aren't any really great high-level docs for tags. [/cli/v8/configuring-npm/package-json#dependencies](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) links to [/cli/v8/commands/npm-dist-tag](https://docs.npmjs.com/cli/v8/commands/npm-dist-tag), which isn't ideal (focused on modification, not usage), though I guess [/cli/v8/commands/npm-dist-tag#purpose](https://docs.npmjs.com/cli/v8/commands/npm-dist-tag#purpose) is nice. Considering expanding [/downloading-and-installing-packages-locally#installing-a-package-with-dist-tags](https://docs.npmjs.com/downloading-and-installing-packages-locally#installing-a-package-with-dist-tags), maybe pulling some content from [/cli/v8/commands/npm-dist-tag#purpose](https://docs.npmjs.com/cli/v8/commands/npm-dist-tag#purpose). Maybe I'll open an issue for that next week.

Reply:
> Tempted to link to https://github.com/npm/node-semver#ranges, that's where [/cli/v8/configuring-npm/package-json#dependencies](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) links to.

npm being one of the package managers, I think linking to SemVer rather than to npm would be preferable to not give a competitive adventage to one rather the others.

Reply:
@aduh95, I'll be out until Monday, feel free to link out to docs for tag and / or range if you think that would be helpful.

Still a little tempted to clarify when Corepack added support for using a tag or range ([Corepack v0.120](https://github.com/nodejs/corepack/releases/tag/v0.12.0), which shipped with [Node.js v18.6.0](https://github.com/nodejs/node/releases/tag/v18.6.0) and [v16.17.0](https://github.com/nodejs/node/releases/tag/v16.17.0)), but not sure if that should be formatted in a special way.